lib/matplotlib/sphinxext/plot_directive.py

```diff
@@ -466,10 +461,7 @@ def run_code(code, code_path, ns=None, function_name=None):
     # Change the working directory to the directory of the example, so
     # it can get at its data files, if any.  Add its path to sys.path
     # so it can import any helper modules sitting beside it.
-    if six.PY2:
-        pwd = os.getcwdu()
-    else:
-        pwd = os.getcwd()
+    pwd = os.getcwd()
     old_sys_path = list(sys.path)
```

Comment:
would rewrite as `sys.path.copy()`, otherwise it's not immediately clear what's happening.